lisa/transformers/kernel_source_installer.py

```diff
@@ -114,23 +119,59 @@ def install(self) -> str:
         # modify code
         self._modify_code(node=node, code_path=code_path)
 
-        self._build_code(node=node, code_path=code_path)
+        config_path = source.get_config_path()
+        self._build_code(node=node, code_path=code_path, kconfig_path=config_path)
 
         self._install_build(node=node, code_path=code_path)
 
         result = node.execute(
-            "make kernelrelease 2>/dev/null", cwd=code_path, shell=True
+            "make kernelrelease 2>/dev/null",
+            cwd=code_path,
+            shell=True,
         )
 
         kernel_version = result.stdout
-        result.assert_exit_code(0, f"failed on get kernel version: {kernel_version}")
+        result.assert_exit_code(
+            0,
+            f"failed on get kernel version: {kernel_version}",
+        )
 
         # copy current config back to system folder.
         result = node.execute(
-            f"cp .config /boot/config-{kernel_version}", cwd=code_path, sudo=True
+            f"cp .config /boot/config-{kernel_version}",
+            cwd=code_path,
+            sudo=True,
         )
         result.assert_exit_code()
 
+        if config_path:
+            # If it is dom0,
+            # Name of the current kernel should be vmlinuz-<kernel version>
+            uname = node.tools[Uname]
+            current_kernel = uname.get_linux_information().kernel_version_raw
+            current_kernel_binary = f"KERNEL_PATH=vmlinuz-{current_kernel}"
+
+            # Copy the binary to /boot/efi from : arch/x86/boot/bzImage
+            source_path = code_path.joinpath("arch/x86/boot/bzImage")
```

Comment:
Does this path apply to all kernels, like arm64, and what's the bzImage standard?

Reply:
bzImage is the image generated while we build the kernel. I think it would remain same but in case of normal flow it would be copied to /boot and make changes in grub and for dom0 we are copying kernel binary to /boot/efi and changing linuxloader.conf

Reply:
Can the arch be gotten from the system? We assume the current machine build the kernel with same arch.

Reply:
As of now it is for x86/x86_64. There is no cross compilation planned as of now. We can get hw platform detail from uname.

Also we would need to install cross compiler dependencies for it.

Reply:
Currently there is no support for arm64 for LinuxDom0 setup, so there is no point of extending LISA to cross compile for ARM64.

Reply:
I changed the source path for kernel binary. make install will execute /sbin/installkernel script and generate new kernel binary at /boot from there we can copy to /boo/efi. 

We dont need to follow path arch/x86/boot/bzImage.

I tried with arm64 as well and our source would remain same if we refer /boot. Please refer the latest commit